src/components/views/rooms/RoomList.js

```diff
@@ -71,6 +71,8 @@ module.exports = React.createClass({
         this.mounted = false;
 
         const cli = MatrixClientPeg.get();
+        const dmRoomMap = new DMRoomMap(cli);
```

Comment:
any reason not to use the shared one? This one will get stale unless you call `start()` on it

Reply:
Oh, only that the RoomList was doing the same thing, although it makes a new one every time it's used.

Reply:
dave's comment about this getting stale looks relevant, still